syntax = "proto3";

package products;

service ProductService {
  rpc GetProductById (ProductRequest) returns (ProductResponse);
}

message ProductRequest {
  int32 id = 1;
}

message ProductResponse {
  int32 id = 1;
  string name = 2;
  string description = 3;
  float cost = 4;
  string status = 5;
  string image_url = 6;
  string attributes = 7;
}
